prow/istio-lint.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Can you hold on this until we know what the new Prow setup and version will look like ? 
My understanding is that they may support a proper yaml configuration - tektoncd or similar. 

I am very uncomfortable with all those scripts - we have a clear goal to use Kind not Mason, 
and there all kind of downloads that don't make any sense - make test is using docker images and 
has download support for stuff that is needed. 

Ok to use Prow - but we should have a clean solution and put all that is common across CI/CDs in make.

Reply:
I can hold off - I just wanted to start learning about "old prow" prior to making the switch to "new prow" so the switch can move quickly.

As a project there is a strong TOC desire to eliminate the dependency on circleci.